src/common/plugin.ts

```diff
@@ -20,6 +20,8 @@ export abstract class Plugin implements IPlugin {
 
   public pluginChannelName?: string;
 
```

Comment:
Can you make pluginChannelName a string | string[] 

that would cut some of this down

Reply:
Done; I'm just not a huge fan of this naming scheme.